docs/TDD_ADOPTION_PLAN.md

```diff
@@ -0,0 +1,255 @@
+# Test-Driven Development Adoption Plan
```

Comment:
This entire file can go away. I don't need documentation on TDD, and going back and adding tests later is definitely *not* TDD anyway.